webrtc.html

```diff
@@ -3153,27 +3153,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
         </dl>
         <dl class="idl" title="dictionary RTCRtpTransceiverInit">
-          <dt>boolean send = true</dt>
-
-          <dd>
-            <p>If true, indicates that
-            the <code><a>RTCRtpTransceiver</a></code>'s <code><a>RTCRtpSender</a></code>
-            will offer to send RTP and send RTP if the remote peer accepts.  If false,
-            indicates that the <code><a>RTCRtpSender</a></code> will not offer to send RTP
-            and will not send RTP (the direction of the <a>media description</a> generated by createOffer
-            will be <code>recvonly</code> or <code>inactive</code>).</p>
-          </dd>
-
-          <dt>boolean receive = true</dt>
-
-          <dd>
-            <p>If true, indicates that
-            the <code><a>RTCRtpTransceiver</a></code>'s <code><a>RTCRtpReceiver</a></code>
-            will offer to receive RTP and receive RTP if the remote peer accepts.  If
-            false, indicates that the <code><a>RTCRtpReceiver</a></code> will not offer to
-            receive RTP and will not receive RTP (the direction of the <a>media description</a> generated by
-            createOffer will be <code>sendonly</code> or <code>inactive</code>).</p>
-          </dd>
+          <dt>RTCRtpTransceiverDirection direction = "sendrecv"</dt>
+          <dd>The direction of the RTP stream.</dd>
```

Comment:
"direction of the RTP stream" is problematic.   In RTP taxonomy, "RTP stream" means something much different than the thing we are describing here. 

A better phrase might be "direction of the RtpTransceiver".